capellambse_context_diagrams/context.py

```diff
@@ -12,6 +12,7 @@
 import typing as t
 
 from capellambse import aird
+from capellambse import diagram as aird_diagram
```

Comment:
Calling this `aird_diagram` is quite misleading, because it has nothing to do with the (also imported) `aird` module. IMO something like `capella_diagram` or `cm_diagram` (or just using `capellambse.diagram` directly) would be better.